parking_permits/models/order.py

```diff
@@ -59,6 +66,61 @@ def validate_order(order_id, user_id):
             raise OrderValidationError("Talpa order is not valid")
 
 
+class SubscriptionValidator:
+    @staticmethod
+    def validate_subscription(user_id, subscription_id, order_id, order_item_id):
+        headers = {
+            "api-key": settings.TALPA_API_KEY,
+            "namespace": settings.NAMESPACE,
+            "user": user_id,
+            "Content-Type": "application/json",
+        }
+        response = requests.get(
+            urljoin(
+                settings.TALPA_ORDER_EXPERIENCE_API,
+                f"subscriptions/get-by-order-id/{order_id}",
+            ),
+            headers=headers,
+        )
+        if response.status_code == status.HTTP_200_OK:
+            response_data = response.json()
+            for subscription in response_data.get("subscriptions"):
```

Comment:
This might be a little clearer if this error checking was done in a separate function e.g. SubscriptionValidator.get_subscription_info(response_data.get("subscriptions"))

Also, I assume we just need the first subscription, but what happens if this list is empty?

Reply:
Updated / fixed these.